src/components/ButtonsGroup/ButtonsGroup.tsx

```diff
@@ -1,25 +1,48 @@
 import React, { useState } from 'react';
-import { FlatList, PressableProps, StyleSheet, View } from 'react-native';
+import { FlatList, StyleSheet, View } from 'react-native';
 import { sizes } from '@magnetis/astro-tokens';
 
 import Button from '@components/Buttons/Button';
 
+import type { PressableProps } from 'react-native';
 import type { ButtonVariant } from '@components/Buttons/types';
 
 type Item = {
   value: string;
   onPressItem: () => void;
 };
 
+export const containerJustifyContentOptions = [
+  'flex-start',
+  'flex-end',
+  'center',
+  'space-between',
+  'space-around',
+  'space-evenly',
+] as const;
+
+export type ContainerJustifyContent = typeof containerJustifyContentOptions[number];
```

Comment:
Não é uma boa ideia reimplementar types do RN. Ao invés disso, melhor você usar os types que já existem:
```suggestion
import { ViewStyle } from 'react-native';
// depois basta utilizar o type onde for necessário
containerJustifyContent?: ViewStyle['justifyContent']
```

Dessa forma, caso novos valores sejam aplicados, não precisamos mudar o código e nem teremos nenhum efeito colateral.